core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -24,10 +24,13 @@ oppia.directive('simpleEditorSidebar', [
         '/pages/exploration_editor/simple_editor_tab/' +
         'simple_editor_sidebar_directive.html'),
       controller: [
-        '$scope', 'EditorModeService', 'SimpleEditorManagerService',
-        'ScrollSyncService', 'QuestionIdService',
-        function($scope, EditorModeService, SimpleEditorManagerService,
-                 ScrollSyncService, QuestionIdService) {
+        'EditorModeService', 'QuestionIdService', '$scope',
```

Comment:
`$scope` should always be first in the import list, then we alphabetize the rest.